package.json

```diff
@@ -121,7 +121,9 @@
     "@types/react": "^16.14.5",
     "babel-preset-react-app": "^9.1.2",
     "react": "16.14.0",
-    "react-dom": "16.14.0"
+    "react-docgen-typescript": "^2.2.2",
```

Comment:
1. Importing the new EntityStore into the reference package caused a build issue with `tsdx` as it was using an old version of typescript
2. Set the typescript resolution to `"typescript": "^4.4.2"`
3. This broke the `docz` build because `docz` uses an old version of `react-docgen-typescript` which using an old version of typescript but now step 2 broke this.
4. Set the react-docgen-typescript resolution to `"react-docgen-typescript": "^2.2.2"`
5. It builds 🎉